tasks/otel.yaml

```diff
@@ -41,6 +41,16 @@ tasks:
       - "{{ .HELM_CMD }} repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts"
       - "{{ .HELM_CMD }} install --values otel-demo-overrides.yaml my-otel-demo open-telemetry/opentelemetry-demo --namespace otel-demo --create-namespace"
 
+  demo-upgrade:
+    desc: Upgrade otel-demo in the local cluster.
+    run: once
+    deps:
+      - task: cluster:start
+      - task: tools:install-helm
+    cmds:
+      - "{{ .HELM_CMD }} repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts"
```

Comment:
not sure, if the `repo add` is required here... as I would assume, before triggering an upgrade someone has done an install before... but I guess it doesn't harm

Reply:
Too late, I didn't see your comment. You are right, it's not needed there. I will remove it.